TESTS/mbedmicro-rtos-mbed/basic/main.cpp

```diff
@@ -19,6 +19,10 @@
 #include "utest/utest.h"
 #include "unity/unity.h"
 
+#if defined(SKIP_TIME_DRIFT_TESTS)
+#error [NOT_SUPPORTED] test not supported
```

Comment:
For this, do you think we should add a dummy test case, and skip the time drifting test by CASE level? rather than block the whole test getting built

Reply:
I was thinking about this, but it looks very strange, so I decided to disable the entire tests.